jupyterhub/values.yaml

```diff
@@ -162,6 +162,7 @@ singleuser:
   extraEnv: {}
   lifecycleHooks:
   initContainers:
+  extraContainers:
```

Comment:
I think both `extraContainers` and `initContainers` should be declared to be `[]` (still Falsy) by default in order to avoid some coalesce warnings by Helm when Helm coalesces the provided value sources.

Reply:
Feel free to add it to both.